gensim/models/hdpmodel.py

```diff
@@ -34,11 +34,11 @@
 from __future__ import with_statement
 
 import logging, time
-import numpy as np
```

Comment:
It will make reviewing this PR much easier if this decorative change wasn't here